src/VisualStudio/CSharp/Impl/VSPackage.resx

```diff
@@ -273,6 +273,7 @@ Insert space before and after binary operators;</value>
     <value>Change completion list settings;Pre-select most recently used member; Completion Lists;
 Show completion list after a character is typed;
 Show completion list after a character is deleted;
+Automatically show completion list in argument lists (experimental);
```

Comment:
Only question is whether we should be putting these all at the bottom of the list to help the translators (rather than having them mirror the UI ordering)? @wli3